eox_core/api/v1/serializers.py

```diff
@@ -83,7 +81,8 @@ class EdxappCourseEnrollmentQuerySerializer(EdxappCourseEnrollmentSerializer):
     Handles the serialization of the context data required to create an enrollemnt
     on different backends
     """
-    username = serializers.CharField(max_length=30)
+    username = serializers.CharField(max_length=30, default="")
```

Comment:
Can we default to `None` instead of an empty string? This way we are not comparing to emtpy strings all over the code

Reply:
Changed to None but FWIW empty strings evaluate to false in python (just like None does)